bigtable/src/main/scala/magnolify/bigtable/BigtableType.scala

```diff
@@ -14,88 +14,112 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-
 package magnolify.bigtable
 
 import java.nio.ByteBuffer
 
-import com.google.protobuf.ByteString
-import magnolia._
-
-import scala.collection._
 import com.google.bigtable.v2.Mutation
 import com.google.bigtable.v2.Mutation.SetCell
 import com.google.cloud.bigtable.data.v2.models.{Row, RowCell}
+import com.google.protobuf.ByteString
+import magnolia._
+import magnolify.shared.Converter
+import magnolify.shims.JavaConverters._
 
 import scala.annotation.implicitNotFound
 import scala.language.experimental.macros
-import magnolify.shims.JavaConverters._
-
-sealed trait BigtableType[T] extends Serializable {
-
-  def from(v: Row, columnFamily: String = BigtableType.DEFAULT_COLUMN_FAMILY_NAME): T
-
-  def to(v: T, columnFamily: String = BigtableType.DEFAULT_COLUMN_FAMILY_NAME,
-               timestamp: Long = 0L,
-               columnQualifier: String = null): Iterable[Mutation]
 
+sealed trait BigtableType[T] extends Converter[T, java.util.List[RowCell], Seq[SetCell.Builder]] {
+  def apply(v: Row, columnFamily: String): T = from(v.getCells(columnFamily))
```

Comment:
Are these good enough? We can reuse the same `BigtableType[T]` for `T` & different column families and timestamp settings. And if there aren't any other arguments we can live with a single method with default `timestampMicros`?

Reply:
I think this is good enough. Can't think of any example when it wouldn't work.